department-of-veterans-affairs/abd-vro
# Move rrd main classes to domain-rrd from app

## What was the problem?

https://github.com/department-of-veterans-affairs/abd-vro/issues/1509

## How does this fix it?[^1]
* First step in removing some of the config files associated with APIs only used by RRD to the `domain-rrd` folder 

## How to test this PR
- Code change only, no logic change, unit/e2e testing is sufficient
- Step 2[^secrel]


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
